doc_src/funcsave.txt

```diff
@@ -8,3 +8,5 @@ funcsave FUNCTION_NAME
 \subsection funcsave-description Description
 
 `funcsave` saves the current definition of a function to a file in the fish configuration directory. This function will be automatically loaded by current and future fish sessions. This can be useful if you have interactively created a new function and wish to save it for later use.
+
+Note that event handlers will not be executed before they are loaded.
```

Comment:
I think it is worth expanding on this.
How about:
"Functions are loaded on-demand, which means that saved functions containing `<a href="index.html#event">event handlers</a>` are not installed when a new shell is started. To activate an event handler for every new shell, add the whole function to your `<a href="index.html#initialization">shell initialization file</a>."